.travis.yml

```diff
@@ -6,6 +6,7 @@ os:
 julia:
   - 0.7
   - 1.0
+  - 1.0.1 
```

Comment:
I think the 1.0 should get actually 1.0.1, according to https://discourse.julialang.org/t/julia-1-0-1-has-been-released/15751 but I haven't verified.

Reply:
That looks correct, I'll pull it out.